src/main/java/org/alliancegenome/curation_api/model/entities/ExperimentalCondition.java

```diff
@@ -83,4 +83,8 @@ public class ExperimentalCondition extends UniqueIdAuditedObject {
 	@JsonView({View.FieldsOnly.class})
 	@Column(columnDefinition = "TEXT")
 	private String conditionFreeText;
+
+	public String getUniqueExperimentId() {
+		return conditionSummary;
+	}
```

Comment:
This is quite confusing as it is basically remapping the name of a variable. This method is not needed, just get the string directly.